docker/network-size-5.yml

```diff
@@ -84,7 +88,7 @@ services:
       RADIXDLT_NODE_KEY: ${RADIXDLT_VALIDATOR_4_PRIVKEY}
     networks:
       - five_nodes
-    ports:
+    ports: # TODO(resolve-during-review): this `core4` looks weird compared to the rest; intentional?
```

Comment:
99.9% not intentional. If you could fix it up as you do this, that would be grand :+1: